cumulus_library/cli.py

```diff
@@ -213,15 +213,23 @@ def get_study_dict(alt_dir_paths: List) -> Optional[Dict[str, PosixPath]]:
     paths = [Path(cli_path, "studies")]
     if alt_dir_paths is not None:
         paths = paths + alt_dir_paths
-    for parent_path in paths:
-        for child_path in parent_path.iterdir():
-            if child_path.is_dir():
-                manifest_studies[child_path.name] = child_path
-            elif child_path.name == "manifest.toml":
-                manifest_studies[parent_path.name] = parent_path
+    for path in paths:
+        found_studies = get_studies_by_manifest_path(path)
+        manifest_studies.update(found_studies)
     return manifest_studies
 
 
+def get_studies_by_manifest_path(path: PosixPath) -> dict:
```

Comment:
This seems like something I would go to `os.walk` for, but PosixPath is doing a lot of the work for you too. :+1:

Reply:
There's def more than one way to do this - I'm making a conscious decision over in this to use pathlib.PosixPath when there's rough equivalence just so there's one pattern for someone to pick up.